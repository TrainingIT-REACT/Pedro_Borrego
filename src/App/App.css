.active {
  text-decoration: none;
  font-weight: bold;
}
html{
  color: white;
}

