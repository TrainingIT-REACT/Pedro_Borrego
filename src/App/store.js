import { createStore, combineReducers } from "redux";

// Reducers
import albums from './reducers/albums';
import user from './reducers/user';
import songs from './reducers/songs';

export default createStore(combineReducers({ albums, user, songs }));