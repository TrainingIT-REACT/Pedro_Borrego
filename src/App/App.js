import React, { Component } from 'react';
import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';


// Componentes que se renderizarán siempre
import TopBarLayout from "./components/topBar"

// Css
import './App.css';

// Componentes que se renderizarán en las distintas rutas

const Album = () => <p>Vista de detalle de album</p>;
const theme = createMuiTheme({
  palette: {
    type: 'dark',
  },
});

class App extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <MuiThemeProvider theme={theme}>
        <div className="App">
          <TopBarLayout />
        </div>
      </MuiThemeProvider>
    );
  }
}

export default App;
