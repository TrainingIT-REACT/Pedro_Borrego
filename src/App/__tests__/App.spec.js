import React from 'react';
import { render } from 'enzyme';

// Componente
import App from '../App';

describe(App, () => {
  it('renders correctly', () => {
    const wrapper = render(<App />);
    expect(wrapper).toMatchSnapshot();
  });
})
