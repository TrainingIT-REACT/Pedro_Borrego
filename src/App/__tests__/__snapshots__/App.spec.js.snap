// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  class="App"
>
  <header
    class="App-header"
  >
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      class="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
